backend/compact-connect/README.md

```diff
@@ -111,8 +111,8 @@ its environment:
    See [Default User Pool Email Settings](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-email.html#user-pool-email-default)
 3) Copy [cdk.context.sandbox-example.json](./cdk.context.sandbox-example.json) to `cdk.context.json`.
 4) At the top level of the JSON structure update the `"environment_name"` field to your own name.
-5) Update the environment entry under `ssm_context.environments` to your own name and your own AWS sandbox account id,
-   and domain name, if you set one up. If you opted not to create a HostedZone, just remove the `domain_name` field.
+5) Update the environment entry under `ssm_context.environments` to your own name and your own AWS sandbox account id (which you can find by following these [these instructions](https://docs.aws.amazon.com/accounts/latest/reference/manage-acct-identifiers.html#FindAccountId)),
+   and domain name, if you set one up. **If you opted not to create a HostedZone, remove the `domain_name` field.**
    The key under `environments` must match the value you put under `environment_name`.
 6) Configure your aws cli to authenticate against your own account.
```

Comment:
Yeah this step is difficult to capture directly in the README since we don't know the developer's authentication strategy (SSO, long term IAM user, etc.) But here is a suggestion which includes AWS documentation for setting up for every potential authentication type

```suggestion
6) Configure your aws cli to authenticate against your own account. There are several ways to do this based on the
   type of authentication you use to login to your account. See the [AWS CLI Configuration Guide](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-quickstart.html).
```

Reply:
Yeah that makes sense! I'd eventually like this to be easy for someone not smart on AWS (me) to be able to complete this step pretty easily, but I think this is a good suggestion for now.